src/main/java/com/xceptance/xlt/report/providers/BasicTimerDataProcessor.java

```diff
@@ -121,6 +121,15 @@ public TimerReport createTimerReport(final boolean generateHistogram)
         // set the counts
         final double count = runTimeStatistics.getCount();
         final long duration = Math.max((getEndTime() - getStartTime()) / 1000, 1);
+        
+        if (count != 0)
+        {
+            timerReport.errorPercentage = ReportUtils.convertToBigDecimal(100 * totalErrors / count);
+        }
+        else
+        {
+            timerReport.errorPercentage = ReportUtils.convertToBigDecimal(0);
+        }
```

Comment:
The percentage calculation is done using integer arithmetic, so the result may be wrong. 

How about creating a helper method like `BigDecimal ReportUtils.calculatePercentage(int count, int totalCount)` that takes care of this subtlety and also includes the check if count is 0? Use this method wherever we calculate error percentages.